AU2/frontends/inquirer_cli.py

```diff
@@ -429,6 +445,104 @@ def render(html_component, dependency_context={}):
             inquirer.Text(name=html_component.identifier, message=html_component.title, default=html_component.default)]
         return inquirer_prompt_with_abort(q)
 
+    elif isinstance(html_component, EditablePseudonymList):
+        # perhaps abstract this?
+        # so that references to pseudonyms and datetimes are contained within attributes of html_component,
+        # rather than being hard-coded here.

```

Comment:
Don't follow - can you elaborate?

Reply:
My thinking here was that fundamentally all that is going on in this interface is editing a list where each entry has multiple parts to its value, so the component doesn't need to be specfically tailored to being the pseudonym editor.
Although, if we do want to reuse the component in another context it probably makes more sense to cross that bridge when we come to it rather than pre-emptively abstracting...

Reply:
I see, thank you. That sounds like a sensible note for future. Perhaps leave a `# TODO: ...` with some of what you have here as it's a good description of the potential improvement.